src/Constants.js

```diff
@@ -1,6 +1,6 @@
 // RPC
 const MaxUTXOs = 255;
-const ReqTimeout = 8000;
+const ReqTimeout = 45000;
 
```

Comment:
My only problem is, do we really want an RPC request to take 45 seconds before failing? In production this seems bad for an end user.

Reply:
https://madhive.atlassian.net/jira/software/c/projects/MP/boards/117/backlog?view=detail&selectedIssue=MP-583&issueLimit=100 should address this by allowing specific timeouts during testing